### PR TITLE
Rewritten Mdns class using the dnssd package

### PR DESCRIPTION
**Pull request by uno20001**
With this update, the Mdns class uses the `dnssd` package to publish two DNS-SD services: an HTTP, and an MQTT instance. The instance name are generated using the `uuid` package. TODO: The published data should be based on some kind of configuration file, etc.

### CODE REVIEW

**Comment by gergof** [comment]
It's awkward to write this in English.
But: multicast-dns is no longer needed, so it can be removed. I wouldn't make a http service advertisment, since we don't have any http server as of now, but we could have a service advertisment for graphql on port 5000.
In the rest it's cool.

File: package.json [hunk 1]
```diff
@@ -42,16 +42,17 @@
     "aedes": "^0.36.0",
     "apollo-server": "^2.3.2",
     "chalk": "^2.4.2",
+    "dnssd": "^0.4.1",
     "graphql": "^14.1.1",
     "graphql-tools": "^4.0.4",
-    "internal-ip": "^4.0.0",
     "lodash": "^4.17.11",
     "mqtt-store": "^2.1.0",
     "multicast-dns": "^7.2.0",
```
Comment (gergof) [inline package.json]: Multicast-dns could be removed

**Comment by uno20001** [comment]
Both of them are there just for testing purposes. As of yet, yes, we don't have an http server, but as far as I know we plan on having one. Publishing the GrapQL UI is pointless in my opinion, because it will never be used by an end user.

**Review by gergof — APPROVED** [review]
OK.
This is a POC. We will change it when the need arises.

**Pull request merged**